features/payment-handler.yml

```diff
@@ -0,0 +1,38 @@
+name: Payment handler
+description: The Payment Handler API lets web applications handle requests for payments directly through the browser user interface, rather than redirecting users to a separate site for payment handling.
```

Comment:
```suggestion
name: Payment handler
description: The payment handler API registers a web application as a payment handler and responds to payment requests in the browser's user interface, rather than redirecting users to a separate site to complete a payment.
```